documentation/modules/ROOT/pages/connectors/postgresql.adoc

```diff
@@ -1577,7 +1575,7 @@ The {prodname} logical decoding plug-ins have been installed and tested on only
 ====
 
 [[postgresql-differences-between-plugins]]
-=== Differences between plug-ins
```

Comment:
FYI - I changed this so that the heading fits on one line in the right panel, instead of breaking. Faster to read.

Reply:
Ok, nice.